examples/cra-kitchen-sink/src/image-storyshots.test.js

```diff
@@ -0,0 +1,10 @@
+import initStoryshots, { imageSnapshotWithOptions } from '@storybook/addon-storyshots';
+import path from 'path';
+
+initStoryshots({
+  suite: 'Image snapshots',
```

Comment:
can you document all of these options and the possible inputs for these in `README.md`?

Reply:
In fact I already documented inside the README the three params for the `imageSnapshot` function. (the function that you assign to the `test` param of _initStoryshots_.)

The "suite" option & all other initStoryshots params are already documented, nothing new from this PR.